book/intro.md

```diff
@@ -272,15 +274,15 @@ of widgets is rarely specified directly.
 [^style-calculation]: For example, when exactly does the browser
 compute HTML element's styles? Any change to the styles is visible to
 all subsequent API calls, so in that sense it applies "immediately."
-But it is better for the browser to delay style re-calculation,
+But it is better for the browser to delay style recalculation,
 avoiding redundant work if styles change twice in quick succession.
 Maximally exploiting the opportunities afforded by declarative
 programming makes real-world browsers very complex.
 
 To me, browsers are where algorithms _come to life_. A browser contains a
 rendering engine more complex and powerful than any computer game; a full
 networking stack; clever data structures and parallel programming techniques; a
-virtual machine, an interpreted language, and a JIT; a world-class security
+virtual machine, an interpreted language, and a just-in-time compiler; a world-class security
```

Comment:
The copyeditor also defined the "JIT" acronym, but I've confirmed that it's not used elsewhere in the book so probably doesn't need to be defined here.